WordPress/Classes/ViewRelated/NUX/Signin2FAViewController.swift

```diff
@@ -75,6 +76,17 @@ import WordPressShared
     // MARK: Configuration Methods
 
 
+    /// Assigns localized strings to various UIControl defined in the storyboard.
+    ///
+    func localizeControls() {
+        verificationCodeField.placeholder = NSLocalizedString("Verification code", comment: "two factor code placeholder")
+
+        let submitButtonTitle = NSLocalizedString("VERIFY", comment: "Title of a button. The text should be uppercase.").localizedUppercaseString
+        submitButton.setTitle(submitButtonTitle, forState: .Normal)
+        submitButton.setTitle(submitButtonTitle, forState: .Highlighted)
```

Comment:
Doesn't seem to work properly for me.  In a new project with a single button dragged onto a storyboard:
```
    override func viewDidLoad() {
        super.viewDidLoad()

        button.setTitle("foo", forState: .Highlighted)
        button.setTitle("bar", forState: [.Normal, .Highlighted])
    }
```
At run time the button's .Normal title is still `Button`, the default.  Its .Highlighted title is properly set to "bar".